src/internal/deploy/assets/enterprise.go

```diff
@@ -41,9 +39,9 @@ func EnterpriseService(opts *AssetOpts) *v1.Service {
 					NodePort: 31651,
 				},
 				{
-					Port:     auth.OidcPort,
+					Port:     OidcPort,
```

Comment:
I don't think we should get into the habit of putting configuration as constants in libraries.  If the auth package is parameterized in terms of an OidcPort, then let it accept it where applicable.  When it's time to wire everything up, we should keep all the arbitrary values in one place.

Reply:
I agree, I think this should be in `serviceenv.Configuration`, but I was trying to avoid additional scope creep in this PR.

Reply:
Apparently this _is_ in the `serviceenv.Configuration`, and this is just for the default value.  Other ports have a hard-coded default value inlined in `assets.go`, so this is actually a step up from that.  I'll rename it to `DefaultOidcPort` and add a field in the asset options, but ugh.